### DISCUSSION TITLE
[2022/05/24] Frontend Night

### DISCUSSION THREAD

**Discussion by jjenko** [wantedly/frontend-night#206] [Archive]
## Policy

- 資料の準備をがんばらない
- 参加・途中退席は自由
- スキップしない

話したい・聞きたいネタを書いてく ✏️

ハッシュタグ [#frontend_night](https://twitter.com/hashtag/frontend_night?src=hash)

Prev: https://github.com/wantedly/frontend-night/discussions/205 

 _Created from https://github.com/wantedly/frontend-night/discussions/179 by [issue-creator](https://github.com/rerost/issue-creator)_

**Comment by chloe463** [comment]
## State of CSS

https://www.youtube.com/watch?v=Xy9ZXRRgpLk
